.github/workflows/release-post-merge.yml

```diff
@@ -34,6 +34,7 @@ on:
         required: true
 jobs:
   create-release:
+    environment: release
```

Comment:
This requires manual configuration on the repository settings, right?
Are you already separating workflow per env?

Reply:
Yes, see here:
https://github.com/mongodb/mongodb-atlas-kubernetes/settings/environments

Envs are NOT per workflow, but per access topic. After that recent task revisiting our actions security one outcome was we wanted to separate testing access from release access: there is NO need to expose release related credentials while running the cloud tests and viceversa. So those should be clearly separated environments.

Apart from that gov tests also seemed to have no need to overlap with the regular tests credentials and the openshift test also seems it can be isolated from all the rest.